### PR TITLE
Update MVC dependencies for routing changes

### PR DESCRIPTION
**Pull request by JamesNK**

### CODE REVIEW

File: src/Mvc/build/dependencies.props [hunk 1]
```diff
@@ -34,8 +34,8 @@
     <MicrosoftAspNetCoreHostingAbstractionsPackageVersion>3.0.0-preview-18606-0099</MicrosoftAspNetCoreHostingAbstractionsPackageVersion>
     <MicrosoftAspNetCoreHostingPackageVersion>3.0.0-preview-18606-0099</MicrosoftAspNetCoreHostingPackageVersion>
     <MicrosoftAspNetCoreHtmlAbstractionsPackageVersion>3.0.0-preview-18606-0099</MicrosoftAspNetCoreHtmlAbstractionsPackageVersion>
-    <MicrosoftAspNetCoreHttpExtensionsPackageVersion>3.0.0-preview-18606-0099</MicrosoftAspNetCoreHttpExtensionsPackageVersion>
-    <MicrosoftAspNetCoreHttpPackageVersion>3.0.0-preview-18606-0099</MicrosoftAspNetCoreHttpPackageVersion>
+    <MicrosoftAspNetCoreHttpExtensionsPackageVersion>3.0.0-preview-18619-0194</MicrosoftAspNetCoreHttpExtensionsPackageVersion>
+    <MicrosoftAspNetCoreHttpPackageVersion>3.0.0-preview-18619-0194</MicrosoftAspNetCoreHttpPackageVersion>
```
Comment (dougbu) [inline src/Mvc/build/dependencies.props]: Everything usually moves in lock-step. Why are many packages left at `18606-0099`?

File: src/Mvc/Directory.Build.props [hunk 2]
```diff
@@ -17,6 +17,7 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <UseRoutingFromFeatureBranch>true</UseRoutingFromFeatureBranch>
     <SharedSourceRoot>$(MSBuildThisFileDirectory)..\Shared\</SharedSourceRoot>
+    <NoWarn>$(NoWarn);NU1605</NoWarn>
```
Comment (dougbu) [inline src/Mvc/Directory.Build.props]: What package are we downgrading? And why?

File: src/Mvc/build/dependencies.props (hunk 1 above)
Comment (JamesNK) [reply src/Mvc/build/dependencies.props]: run upgrade deps is broken and won't be fixed before MVC moves to `<reference>`. I could change everything to `18619-0194` but there would be depedencies with other versioning schemes that aren't the latest.

Think of this as a temporary thing until MVC is updated and does away with package references all together.

File: src/Mvc/Directory.Build.props (hunk 2 above)
Comment (JamesNK) [reply src/Mvc/Directory.Build.props]: The deps of 18619-0194 are being downgraded to 18606-0099

Comment (dougbu) [reply src/Mvc/Directory.Build.props]: If the build works when you do the equivalent of `s/18606-0099/18619-0194/g`, do that. Don't worry about the other versioning schemes unless they cause NU1605.

If additional NU1605 warnings come up, do what they say to make this build.

**Comment by JamesNK** [comment]
🆙 📅

**Review by natemcmaster — APPROVED** [review]

**Pull request merged**